.gitignore

```diff
@@ -18,3 +18,4 @@ pip-delete-this-directory.txt
 .dpl
 db.sqlite
 log/
+wsgi/static/
```

Comment:
That's a good idea but should not be in this PR :)
(We should do a cleanup pass on the repo I think)